modules/ROOT/pages/snippets/trust_gpg.adoc

```diff
@@ -0,0 +1,21 @@
+By default, some operating systems do not trust the GPG key for the {productname} client tools.
+The clients can be successfully bootstrapped without the GPG key being trusted.
+However, you will not be able to install new client tool packages or update them until the keys are trusted.
+
+
+
+.Procedure: Trusting GPG Keys on Clients
+. On the {productname} Server, at the command prompt, check the contents of the [path]``/srv/www/htdocs/pub/`` directory.
+    This directory contains all available public keys.
+    Take a note of the key that applies to the client you are registering.
```

Comment:
This recommendation wonders me a little bit. Does it really depend on the client?  Later on, we have `uyuni-`something as an example, but "uyuni" is the server.

Reply:
Good question. This has been like this for some time. Perhaps we need @mcalmer to weigh in?